src/KubeLibrary/KubeLibrary.py

```diff
@@ -124,6 +166,7 @@ def reload_config(self, kube_config=None, context=None, api_url=None, bearer_tok
                 config.load_kube_config(kube_config, context)
             except TypeError:
                 logger.error('Neither KUBECONFIG nor ~/.kube/config available.')
+        self.dynamic_client = dynamic.DynamicClient(client.ApiClient(configuration=client.Configuration().get_default_copy()))
```

Comment:
Notice that above we are building up configuration object, you should use same here. Also in _add_api we are handling disabling cert validation, sth similar should be applied to dynamic client.

Reply:
You're right. I will fix this